src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -405,6 +405,31 @@ int NonClientIslandWindow::_GetResizeHandleHeight() const noexcept
     return HTCAPTION;
 }
 
+// Method Description:
+// - Gets the difference between window and client area size.
+// Arguments:
+// - dpi: dpi of a monitor on which the window is placed
+// Return Value
+// - The size difference
+SIZE NonClientIslandWindow::GetTotalNonClientExclusiveSize(UINT dpi) const noexcept
+{
+    RECT islandFrame{};
+
+    // If we failed to get the correct window size for whatever reason, log
+    // the error and go on. We'll use whatever the control proposed as the
+    // size of our window, which will be at least close.
+    LOG_IF_WIN32_BOOL_FALSE(AdjustWindowRectExForDpi(&islandFrame, WS_OVERLAPPEDWINDOW, false, 0, dpi));

```

Comment:
This should be either querying the HWND for its creation flags or using the same creation flags as were given to `CreateWindowEx` wherever that is (if it doesn't change.)

Reply:
That's #3181

Reply:
OK, again, sorry. Will go read it.